src/background.js

```diff
@@ -1047,6 +1052,18 @@ function onMessageHandler(request, sender, callback) {
 		promoModals.turnOffPromos();
 		return false;
 	}
+	if (name === 'debug_information') {
```

Comment:
See review notes. We want to expose limited data here.

Reply:
done, limited to accountEvents, browserInfo, extensionInfo, and user.syncedUserSettings